<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../QtLib/QtLib/Resources/resources.qrc">
    <normaloff>:/icon/png_64/video-television-2.png</normaloff>:/icon/png_64/video-television-2.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>10</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QFrame" name="frameLeft">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QCommandLinkButton" name="cmdWelcome">
           <property name="text">
            <string>Welcome</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="cmdNodeList">
           <property name="text">
            <string>NodeList</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="cmdGroupList">
           <property name="text">
            <string>GroupList</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="cmdReport">
           <property name="text">
            <string>Report</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton_3">
           <property name="text">
            <string>Map</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton_4">
           <property name="text">
            <string>Live Monitor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Test</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>12</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <widget class="FormWelcome" name="tab">
         <attribute name="title">
          <string>Welcome</string>
         </attribute>
        </widget>
       </widget>
      </widget>
      <widget class="DLogger" name="logger" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionLogger"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="actionLogout"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="DStatusBar" name="statusBar"/>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="../../QtLib/QtLib/Resources/resources.qrc">
     <normaloff>:/icon/png_16/system-log-out.png</normaloff>:/icon/png_16/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset resource="../../QtLib/QtLib/Resources/resources.qrc">
     <normaloff>:/icon/png_16/network-disconnect.png</normaloff>:/icon/png_16/network-disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../QtLib/QtLib/Resources/resources.qrc">
     <normaloff>:/icon/png_16/system-shutdown.png</normaloff>:/icon/png_16/system-shutdown.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLogger">
   <property name="text">
    <string>Logger</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DLogger</class>
   <extends>QWidget</extends>
   <header>Inc/Logger</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DStatusBar</class>
   <extends>QStatusBar</extends>
   <header location="global">Inc/StatusBar</header>
  </customwidget>
  <customwidget>
   <class>FormWelcome</class>
   <extends>QWidget</extends>
   <header>formwelcome.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../QtLib/QtLib/Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
